conf/messages.en

```diff
@@ -1,10 +1,9 @@
 feedback=This is a new service - your {0} will help us to improve it.
 feedback.link=feedback
 
-service.name=Plastic Packaging Tax
+service.name=Register for Plastic Packaging Tax
```

Comment:
Are there any knock-on effects of changing this?

Reply:
Haven't come across any but can talk about this if you like.

Reply:
Sounds like @SteveSugden is also convinced that this will not cause a problem.